meetings/2020-02-11.md

```diff
@@ -0,0 +1,29 @@
+
+## 2020-02-11 Summit Planning Meeting
+
+**Present:** Manil, Ruy, Christian, Rachel, Jory, Joe, Wes
+
+**Agenda**
+
+1. **Venue & Date Updates**
+    1. 2 Member spaces reported back - not available. Looking into others.
+    2. LF Events team looking into whether there is any space that can be recaptured from the conference events on thurs-fri
+    3.  Idea from Wes: Capital Factory. 
+3. **Registration**
+    1. Registration to launch with OpenJS World registration - expected by 2.21 if not sooner (ASAP)
+    2. Registration form questions were reviewed by the group in Slack
+    3. Nominally set price for $15 (exceptions available)
+        * will plan to have registration process, namely to get a firm number of attendees. To be discussed at a future meeting.
+4. **Summit Programming**
+    1. Research Question: Would your project be interested in holding its own collab summit on the same day in the same city as the OpenJSF Collab Summit?
+        12. Answer was generally yes! But we may not have time to pull that off for June 2020. Discussion / Interest in having project 'tracks' seems possible.[https://docs.google.com/spreadsheets/d/1TjTw5An3eTLP735cu7mNEFAoyd19p1ua54HhH57YIcI/edit#gid=0](https://docs.google.com/spreadsheets/d/1TjTw5An3eTLP735cu7mNEFAoyd19p1ua54HhH57YIcI/edit#gid=0)
+    12. Discussion about friendliness/accessibility to new project contributors.
+        * would like to schedule a 'new community member orientation' track - getting started in open source, getting to know the Foundation community, etc.
+        * Could include brief sessions from project maintainers to introduce their project
+        * Encourage maintainers who are speaking to do a 'getting started' session after their talk
+        * emphasize workshops, learning
+        * Could these run during the main conference?
+    3. Discussion about meta-topics - organizing session calls around cross-project relevant themes (best practices, security)
+    4. Idea: Put calls out for session topics; attempt to have majority in by late April & plan to fill in from there.
+    5. Idea: Don't require session facilitator/leader in order to encourage people to surface discussions they would like to have
```

Comment:
From previous summits experience, having a facilitator for most session means that the sessions and topics are prepared up-front and curated. I highly discourage removing this.

We might want to keep a few slots as "unconference" / "open" if we want to have more spontaneous discussions

Reply:
@mcollina since this is just the meeting notes, I'll pull this out into a separate issue so we can continue the thread :D

Reply:
Captured in #250